ecosystem/typescript/sdk/examples/typescript/package.json

```diff
@@ -5,6 +5,7 @@
   "main": "index.js",
   "scripts": {
     "bcs_transaction": "ts-node bcs_transaction.ts",
+    "multisig_account": "ts-node multisig_account.ts",
```

Comment:
Could you also add this to `test` on line 12? That way this new example will be run as part of the examples CI.

Reply:
Updated